WordPress/src/main/java/org/wordpress/android/util/publicdata/WordPressPublicData.kt

```diff
@@ -21,4 +21,18 @@ class WordPressPublicData @Inject constructor() {
         Wasabi.type -> Wasabi.value
         else -> throw IllegalArgumentException("Failed to get Jetpack package ID: build flavor not found.")
     }
+
+    fun currentPackageVersion() = packageManagerWrapper.getPackageInfo(currentPackageId())?.versionName
+
+    fun nonSemanticPackageVersion(): String? {
+        val rawVersion = currentPackageVersion()
+
+        // Clean app semantic versioning info. E.g 21.2-rc-3 turns to 21.2
+        val wordPressVersion = rawVersion?.split("-")?.getOrNull(0) ?: rawVersion ?: ""
```

Comment:
I was wondering if it makes sense to use RegEx here too? I don't see any problem with this approach, I just wanted to suggest in case you find it more readable.

e.g.
```
    val rawVersion = "21.2-rc-3"
    val majorMinorRegex = "^(\\d*)(\\.(\\d*))".toRegex()
    val wordPressVersion = 
    majorMinorRegex.find(rawVersion) // 21.2
```

Reply:
That's indeed cleaner. Thank you for the suggestion @RenanLukas 🙇 
I've added this with 02fec9c586c50847ce4a71db76869562f2a707e4